params/protocol_params.go

```diff
@@ -43,6 +47,15 @@ var (
 	MinGasLimit     = big.NewInt(5000)                  // Minimum the gas limit may ever be.
 	GenesisGasLimit = big.NewInt(4712388)               // Gas limit of the Genesis block.
 	TargetGasLimit  = new(big.Int).Set(GenesisGasLimit) // The artificial target
+	HardGasLimit    *big.Int                            // Hard gas limit never to allow going over
```

Comment:
Would put all of these in the miner package since it's a miner strategy params, not consensus protocol.